### PR TITLE
Integrate pallet-circuit-execution-delivery into runtime

### PR DESCRIPTION
**Pull request by beqaabu**
In this PR I added `pallet-circuit-execution-delivery` into runtime

### CODE REVIEW

File: circuit/src/runtime/src/lib.rs [hunk 1]
```diff
@@ -498,6 +502,173 @@ impl pallet_xdns::Config for Runtime {
...
+        fn_name: &str,
+        _input: &[u8],
+        _escrow_account: &<Runtime as frame_system::Config>::AccountId,
+        _requested: &<Runtime as frame_system::Config>::AccountId,
+        _callee: &<Runtime as frame_system::Config>::AccountId,
+        _value: BalanceOf<Runtime>,
+        _gas_meter: &mut volatile_vm::gas::GasMeter<Runtime>,
+    ) -> DispatchResult {
+        match (module_name, fn_name) {
+            ("Weights", "complex_calculations") => {
```
Comment (MaciejBaj) [inline circuit/src/runtime/src/lib.rs:524]: Remove for now

File: circuit/src/runtime/src/lib.rs [hunk 2]
```diff
@@ -305,13 +309,13 @@ parameter_types! {
...
 }
 
-pub type PolkadotLikeGrandpaInstance = pallet_bridge_grandpa::Instance1;
-impl pallet_multi_finality_verifier::Config<PolkadotLikeGrandpaInstance> for Runtime {
-    type BridgedChain = bp_polkadot_core::PolkadotLike;
-    type MaxRequests = MaxRequests;
-    type WeightInfo = pallet_multi_finality_verifier::weights::GatewayWeight<Runtime>;
-    type HeadersToKeep = HeadersToKeep;
-}
+// pub type PolkadotLikeGrandpaInstance = pallet_bridge_grandpa::Instance1;
```
Comment (MaciejBaj) [inline circuit/src/runtime/src/lib.rs:312]: Leve Instance1 of bridge connected

File: circuit/src/runtime/src/lib.rs [hunk 3]
```diff
@@ -498,6 +502,173 @@ impl pallet_xdns::Config for Runtime {
...
+
+impl pallet_circuit_execution_delivery::Config for Runtime {
+    type Event = Event;
+    type Call = Call;
+    type AccountId32Converter = AccountId32Converter;
+    type ToStandardizedGatewayBalance = CircuitToGateway;
+}
+
+type Blake2ValU64BridgeInstance = ();
+// type Blake2ValU32BridgeInstance = pallet_multi_finality_verifier::Instance1;
```
Comment (MaciejBaj) [inline circuit/src/runtime/src/lib.rs:596]: Leave 4 active

**Review by MaciejBaj — CHANGES_REQUESTED** [review]

File: circuit/src/runtime/src/lib.rs (hunk 1 above)
Comment (beqaabu) [reply circuit/src/runtime/src/lib.rs]: fixed in the following commit

File: circuit/src/runtime/src/lib.rs (hunk 3 above)
Comment (beqaabu) [reply circuit/src/runtime/src/lib.rs]: fixed in the following commit

File: circuit/src/runtime/src/lib.rs (hunk 2 above)
Comment (beqaabu) [reply circuit/src/runtime/src/lib.rs:312]: fixed in the following commit

**Review by zannis — APPROVED** [review]

**Review by MaciejBaj — APPROVED** [review]

**Pull request merged**